packages/shared/pkg/schema/access_token.go

```diff
@@ -15,7 +15,11 @@ type AccessToken struct {
 
 func (AccessToken) Fields() []ent.Field {
 	return []ent.Field{
+		field.UUID("unique_id", uuid.UUID{}).StorageKey("id").Immutable().Unique().Annotations(entsql.Default("gen_random_uuid()")),
 		field.String("id").Unique().StorageKey("access_token").Immutable().SchemaType(map[string]string{dialect.Postgres: "text"}),
```

Comment:
I think you can do it directly like this, otherwise it will be confusing 
```suggestion
		field.UUID("id", uuid.UUID{}).Immutable().Unique().Annotations(entsql.Default("gen_random_uuid()")),
		field.String("access_token").Immutable().SchemaType(map[string]string{dialect.Postgres: "text"}),
```